### PR TITLE
add clickvisual intro

### PR DESCRIPTION
**Pull request by kl7sn**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by kl7sn** [comment]
@alexey-milovidov

**Comment by sevennt** [comment]
It's an awosome project, is there anyone review this PR?

**Comment by alexey-milovidov** [comment]
Alexey Milovidov, [6/4/22 12:54 AM]
Is it a good tool? https://github.com/ClickHouse/ClickHouse/pull/37635

flynn, [6/4/22 7:13 AM]
I think it’s a awesome tool.

File: docs/ru/interfaces/third-party/gui.md [hunk 1]
```diff
@@ -147,6 +147,18 @@ sidebar_label: "Визуальные интерфейсы от сторонни
 -   Резервное копирование и восстановление.
 -   Управление доступом на основе ролей.
 
+### ClickVisual {#clickvisual}
+
+[ClickVisual](https://clickvisual.gocn.vip/) ClickVisual — это легкодоступная платформа для запросов, анализа, вызовов в полицию.
```
Comment (alexey-milovidov) [inline docs/ru/interfaces/third-party/gui.md:152]: I leave it as is for a while :laughing:

**Pull request merged**